linodecli/output.py

```diff
@@ -144,3 +147,23 @@ def _select_json_elements(self, keys, json):
                 if v:
                     ret[k] = v
         return ret
+
+    def _markdown_output(self, header, data, columns, to):
+        """
+        Pretty-prints data in a Markdown-formatted table.  This uses github's
+        flavor of Markdown
+        """
+        content = []
+
+        if isinstance(columns[0], str):
+            content=data
+        else:
+            for model in data:
+                content.append([attr.render_value(model, colorize=False) for attr in columns])
+
+        if header:
+            print('| ' + ' | '.join([str(c) for c in header]) + ' |')
+            print('|---' * len(header) + '|')
+
+        for row in content:
+            print('| ' + ' | '.join([str(c) for c in row]) + ' |')
```

Comment:
Should be concerned with escaping (`\_`, `\|`)?

Underscore is definitely possible in common fields.

Breaking characters, like pipes and newlines, could cause edge-case hardships.

I wouldn't hold this PR up on either of those.